packages/zmarkdown/config/latex/index.js

```diff
@@ -11,7 +11,9 @@ const rebberConfig = {
       'sCustomBlock',
       'secretCustomBlock',
     ]),
-    heading: require('rebber-plugins/dist/preprocessors/headingVisitor'),
+
+    heading:          require('rebber-plugins/dist/preprocessors/headingVisitor'),
+    quizzCustomBlock: require('./prepare-quizz'),
```

Comment:
My point is : logically rebber-pluggin is a package with all the stringifiers to use to translate our custom nodes into latex.

Here, we do have a specific code for zmd, not for the translation of remark-custom-block.

Reply:
`rebber-plugin` is a bit of a mess anyway, we already have these kind of preprocessors in it.
Look at `spoilerFlatten` for instance, it's ZMd-specific but in `rebber-plugins` anyway.

To me, `rebber-plugin` gathers everything needed on ZMd-side for `rebber`; altough it may be debatable, I don't think it's such a bad thing.

Reply:
Ok, I will move it this afternoon